controllers/socket/socket.go

```diff
@@ -128,4 +128,39 @@ func SocketInit(so socketio.Socket) {
 		return string(bytes)
 	})
 
+	so.On("readyPlayer", func(jsonstr string) string {
+		js, _ := simplejson.NewFromReader(strings.NewReader(jsonstr))
```

Comment:
check for errors. If someone sends an evil message that's not json, this will return nil and error, you'll call Get("steamid") on nil and the server will crash